internal/webhook_dispatcher/webhookdispatcher.go

```diff
@@ -20,11 +20,23 @@ import (
 
 // WebhookEvent Webhook event wrapper
 type WebhookEvent struct {
```

Comment:
Revert changes for fixing redeliver webhook as it is not related to current PR. This needs to be raised as a separate issue and fixed in different PR.